packages/react/.storybook/Container.js

```diff
@@ -1,6 +1,7 @@
-import React, { Component } from 'react';
+import './styles';
```

Comment:
my comments got lost during the review but just wondering, does the import need to happen in this order?

Reply:
@emyarod this was more-so a practice that I did in product where you want style imports to be before javascript imports otherwise styles can load in an unintended order. Can definitely change back, just was a habit lol.

---

The specific case I'm thinking about is imagine three components A, B, C with styles `a.scss`, `b.scss`, and `c.scss`. If they are authored like:

```js
// a.js
import './b';
import 'a.scss';

// b.js
import './c';
import 'b.scss';

// c.js
import 'c.scss';
```

Then you'd end up with import order
- `import 'c.scss';`
- `import 'b.scss';`
- `import 'a.scss';`

Which was reverse of what I would expect. This could definitely not be the case anymore so could go in and change if not 👍

Reply:
since it doesn't affect anything in this case I'm fine with it either way, was just curious because of the diff